README.md

```diff
@@ -4,24 +4,24 @@ gcsim is a Monte Carlo simulation tool used to model team dps. It allows for the
 
 # Getting started
 
-You can download the latest build [here](https://github.com/genshinsim/gcsimui/releases). We are also beta testing a new webapp at [https://beta.gcsim.app](https://beta.gcsim.app).
+You can download the latest build [here](https://github.com/genshinsim/gcsim/releases). You can also use the webapp: [https://gcsim.app](https://gcsim.app).
 
 ## Project status
 
-The project is still currently under heavy development. Not every character/weapon/artifacts have been implemented. There are also many quality of life improvements that can be made. Currently this project is developed by one person (me) and as such development speed is not as fast as it can be. If you wish to contribute, please see below for ways you can help out.
+The project is still under development. While every character up until 2.8 has been implemented with proper frames and hitlag, there are still improvements that can be made, which you can find in our [issues](https://github.com/genshinsim/gcsim/issues?q=is%3Aopen+is%3Aissue) and [discussions](https://github.com/genshinsim/gcsim/discussions). Currently this project is developed by one person (me) and as such development speed is not as fast as it can be. If you wish to contribute, please see below for ways you can help out.
```

Comment:
SInce you're already editing this, want o maybe just get rid of this part? `Currently this project is developed by one person (me) and as such development speed is not as fast as it can be. If you wish to contribute, please see below for ways you can help out.`

Reply:
Done